Adyen/UI/Views/NetworkImageView.swift

```diff
@@ -37,7 +37,18 @@ open class NetworkImageView: UIImageView {
     
     private var dataTask: URLSessionDataTask?
     
+    fileprivate func setImage(_ image: UIImage) {
+        DispatchQueue.main.async {
+            self.image = image
+            self.dataTask = nil
+        }
+    }
+
     private func loadImage(from url: URL) {
+        if url.lastPathComponent.hasPrefix("applepay") {
+            return setImage(UIImage(named: "apple_pay", in: Bundle.coreInternalResources, compatibleWith: nil)!)
+        }
```

Comment:
why do we use a local version? that means if Apple changes its icon specs, we would have to update the SDK, using the remote one is better because its easier to change, right?